.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,61 @@
+name: CI-PR
+on:
+  pull_request:
+    branches: [master]
+concurrency:
+  group: ci-pr-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  ci-pr:
+    name: CI-PR
+    needs: [test, msrv]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Done
+        run: exit 0
+  test:
+    name: Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        rust: [stable]
```

Comment:
> That doesn't automatically force us to update our MSRV but we aren't prepared with a strategy for when that happens

We need to know when that happens. The main reasons I want msrv here are:

1. stable is being tested on master.
2. MSRV is most important thing people would miss when contributing because as you said, they would probably develop on stable and thus getting it work on stable.

Reply:
Moved